faster_raster_test.go

```diff
@@ -204,13 +206,25 @@ func Test_Processing(t *testing.T) {
 			err := raster.Run()
 			So(err, ShouldBeNil)
 
-			img, err := raster.GeneratePageImage(2, 1024, 0)
+			img, err := raster.GeneratePageImage(context.Background(), 2, 1024, 0)
 
 			So(err, ShouldBeNil)
 			So(img, ShouldNotBeNil)
 			raster.Stop()
 		})
 
+		Convey("returns an error when the image raster request is cancelled", func() {
+			err := raster.Run()
+			defer raster.Stop()
+			So(err, ShouldBeNil)
+
+			ctx, doneFunc := context.WithTimeout(context.Background(), 1*time.Millisecond)
+			defer doneFunc()
+
+			_, err = raster.GeneratePageImage(ctx, 2, 1024, 0)
+			So(err, ShouldEqual, ErrRasterTimeout)
+		})
+
```

Comment:
Need a test here where the passed-in context times out as well.

Reply:
I'll see about adding some more tests in the future and maybe another refactor pass. For now, I'm returning a timeout error even when the context is cancelled, but I should probably return a cancelled error.